tldr/cross_file_calls.py

```
@@ -3939,3 +4062,231 @@ def _build_php_call_graph(
+        "def", "defp", "defmodule", "defmacro", "defmacrop", "defguard", "defguardp",
+        "defstruct", "defprotocol", "defimpl", "defexception", "defdelegate",
+        "defoverridable", "defcallback", "defmacrocallback",
+        "alias", "import", "use", "require",
+        "if", "unless", "case", "cond", "with", "for", "try", "receive",
+        "raise", "reraise", "throw", "exit",
+        "quote", "unquote", "unquote_splicing",
+        "super", "__MODULE__", "__DIR__", "__ENV__", "__CALLER__",
+    }
+
+    # Pass 1: Collect all defined function names
+    def collect_definitions(node):
```

Ruff (0.15.5)

[warning] Missing return type annotation for private function `collect_definitions`

Add return type annotation: `None`

(ANN202)